app/controllers/wishlist_items_controller.rb

```diff
@@ -23,7 +23,8 @@ def create
     @wishlist_item = @wishlist.wishlist_items.create!(
       item: item,
       quantity: params[:qty],
-      staff_message: params[:staff_message])
```

Comment:
Nice fix!

Looking at this controller method, one of these days, we need to extract all that logic into a form object. There's a lot going on there...